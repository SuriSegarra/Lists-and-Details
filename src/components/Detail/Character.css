.Character {
    height: 100%;
    background-color: whitesmoke;
    border: black 1px solid;
    border-radius: 5%;
    overflow: auto;
  
    img {
      display: block;
      margin: auto;
      width: 100%;
    }
  
    p {
      padding: 1rem 2rem;
    }
  }