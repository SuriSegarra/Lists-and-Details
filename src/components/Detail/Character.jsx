import React from 'react';
import PropTypes from 'prop-types';
import styles from './Character.css';

const Character = ({ name, image }) => (
  <div className={styles.Character}>
    <h2>{name}</h2>
    <img src={image}/>
  
  </div>
);

Character.propTypes = {
  name: PropTypes.string.isRequired,
  image: PropTypes.string.isRequired
};

export default Character;
