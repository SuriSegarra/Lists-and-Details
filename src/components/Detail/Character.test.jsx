import React from 'react';
import { shallow } from 'enzyme';
import Character from './Character';

describe('Character component', () => {
  it('renders Character', () => {
    const wrapper = shallow(<Character 
      name="Arnold Shortman" 
      image="https://vignette.wikia.nocookie.net/heyarnold/images/f/f5/Arnold.png/revision/latest?cb=20181027162333"/>);
    expect(wrapper).toMatchSnapshot();
  });
});