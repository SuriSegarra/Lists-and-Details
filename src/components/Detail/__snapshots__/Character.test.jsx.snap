// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Character component renders Character 1`] = `
<div
  className="Character"
>
  <h2>
    Arnold Shortman
  </h2>
  <img
    src="https://vignette.wikia.nocookie.net/heyarnold/images/f/f5/Arnold.png/revision/latest?cb=20181027162333"
  />
</div>
`;
